scripts/inference/README.md

```diff
@@ -236,3 +258,12 @@ PYTHONPATH=/mnt/work/FasterTransformer python scripts/inference/run_mpt_with_ft.
     --ckpt_path mpt-ft-7b/1-gpu --lib_path /mnt/work/FasterTransformer/build/lib/libth_transformer.so \
     --sample_input_file prompts.txt --sample_output_file output.txt
 ```
+
+## Running MPT with TensorRT-LLM
+
+MPT-like architectures can be used with the NVIDIA's [TensorRT-LLM](https://github.com/NVIDIA/TensorRT-LLM) library for language model inference. To do so, follow the instructions in the [examples/mpt](https://github.com/NVIDIA/TensorRT-LLM/tree/v0.6.1/examples/mpt) directory for the most recent release, which will show you how to:
+
+1. Convert an MPT HuggingFace checkpoint into the FasterTransformer format.
+2. Build a TensorRT engine with the FasterTransformer weights
+
+Using this engine, you can utilize TensorRT-LLM for fast inference. If you would like to use TensorRT-LLM as an end-to-end solution for an inference service, you can utilize the built engine with an NVIDIA Triton server backend: an example server can be found in [this repository](https://github.com/triton-inference-server/tensorrtllm_backend/tree/v0.6.1) accompanying the most recent release.
```

Comment:
was "built engine" supposed to be "built-in engine"?

Reply:
I'd rephrase it as "built TRT engine". Also, here again we should drop "most recent release" as suggested by Daniel above.

Reply:
@linden-li can you pls make the suggested changes here? also, update TRT LLM link to v0.7.1?